src/App.tsx

```diff
@@ -1,12 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { GoodsList } from './Components/GoodsList/GoodsList';
```

Comment:
but you have components instead of Components. And there one file
Make sure that you push all changes